source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: "Understanding unknownProperty in Ember"
+comments: true
+author:
+  name: Santiago Ferreira
+  email: santiago.ferreira@wyeworks.com
+  twitter_handle: san650
+  github_handle: san650
+  image: /images/team/santiago-ferreira.jpg
+  description: Developer working with Ruby, JavaScript and Ember.
+---
+
+The Ember object model provides support for dynamically resolving properties in
```

Comment:
Not sure 100% `dynamically resolving properties` -> `resolving properties dynamically `